### PR TITLE
boards: Makefile cleanup for MSP430 based boards

### PR DESCRIPTION
**Pull request by haukepetersen**
Travis is complaining for #3762, as the `tests/driver_adt7310` fails for the `z1`. The reason is, that the `z1` is compiled with `-std=c99`, while all other msp430 boards are compiled without this option...

This made me clean up all the msp430 board's Makefiles, to make sure they are using the same compiler options... Now, with `-std=gnu99` (as we are using for cortexm boards) everything compiles smoothely...

### CODE REVIEW

**Comment by OlegHahm** [comment]
@thomaseichinger, can you take care (or reassign)? I'm mostly busy with gnrc stuff currently.

**Comment by thomaseichinger** [comment]
@OlegHahm  I'm on it already.

**Comment by OlegHahm** [comment]
thx

File: boards/chronos/Makefile.include [hunk 1]
```diff
@@ -1,25 +1,14 @@
...
+export OFLAGS = -O ihex
 export FLASHER = mspdebug
-export USEMODULE += msp430_common
 export FFLAGS = rf2500 "prog $(HEXFILE)"
-export OFLAGS = -O ihex
-
-export UNDEF += $(BINDIR)msp430_common/startup.o
 
+# pull in dependencies
 include $(RIOTBOARD)/$(BOARD)/Makefile.dep
```
Comment (thomaseichinger) [inline boards/chronos/Makefile.include]: This file is deleted.

**Comment by thomaseichinger** [comment]
msb-430 is missing `sht11-board.h`.

**Comment by haukepetersen** [comment]
fixed `msb-430` and `chronos`

**Comment by thomaseichinger** [comment]
ACK & Go

**Pull request merged**